### DISCUSSION TITLE
Update-DbaInstance] Update failed: Update failed with exit code -2067922934

### DISCUSSION THREAD

**Discussion by hurricanekal** [dataplat/dbatools#8625] [General] [answered]
Dear Support
I am trying to patch one of my SQL server 2017 database servers running on Windows 2019 standard edition; and im using the script below:

#Update the Build Reference so DBATOOLS is aware of the latest SP/CU versions
#Set the versions to whatever version you're using
Get-DbaBuildReference -MajorVersion 2017 -ServicePack RTM -Update
 
#create a list of servers that you want to patch
$ServerList = 'XXXXXXX'
 
#create a credential to pass in to the Update-DbaInstance command; this will prompt for your password
$cred = Get-Credential DOMAIN\userid
 
#Set the version that you want to update to
$version = '2017RTMCU30'
 
#Start Patching! The -Restart option will allow it to restart the SQL Server as needed
Update-DbaInstance -ComputerName $ServerList -Path 'E:\Software\SQLServer\2017\RTM\' -Credential $cred -Version $version -Restart

However when i run this command i get the error below but i cant find where dbatools stores its log files to capture errors.

Update-DbaInstance] Update failed: Update failed with exit code -2067922934

Please advise 

K.

**Comment by hurricanekal** [comment, marked as answer]
While testing the function i figured that the SQL patch file gets extracted to a certain folder and from there i copied and the extracted folder and tried to run the patching manually. Voila i saw the error; see attached image.
There was a new GPO which was implemented in our company and as soon as we disabled it the update worked beautifully.
If anyone else sees the errors i witnessed.
![DBAUpdate_SQLError](https://user-images.githubusercontent.com/116997773/198989271-5833cca2-627b-42dc-b1b4-f10926fd34a0.png)

**Reply by potatoqualitee** [reply]
Oh, awesome! Thank you so much for the followup 🥳
